### PR TITLE
feat-made cart and wishlist context

### PR DESCRIPTION
**Pull request by therajatg**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,9 +1,5 @@
 import "./App.css";
-import { Cart } from "./frontend/pages/cart/cart";
-import { Home } from "./frontend/pages/home/Home";
-import { Signup } from "./frontend/pages/signup/signup";
-import { Wishlist } from "./frontend/pages/wishlist/Wishlist";
-import { Login } from "./frontend/pages/login/login";
```
Comment (agrawalshanu708) [inline src/App.js:6]: import from the index of seperate folder, dont hardcode it

**Review by agrawalshanu708 — COMMENTED** [review]
good to merge

**Review by prathmeshjagtap — APPROVED** [review]
code is well structured and clean

File: src/App.js (hunk 1 above)
Comment (therajatg) [reply src/App.js:6]: ok, will do it.

**Pull request merged**